monix-reactive/shared/src/main/scala/monix/reactive/Observable.scala

```diff
@@ -1751,6 +1752,28 @@ abstract class Observable[+A] extends Serializable { self =>
   final def map[B](f: A => B): Observable[B] =
     self.liftByOperator(new MapOperator(f))
 
+  /** Applies a function that you supply to each item emitted by the
+    * source observable, where that function returns a sequence of elements, and
+    * then concatenating those resulting sequences and emitting the
+    * results of this concatenation.
+    *
+    * ==Example==
+    * {{{
+    *   Observable(1, 2, 3).mapConcat( x => Seq(x, x * 10, x * 100))
```

Comment:
```suggestion
    *   Observable(1, 2, 3).mapConcat( x => List(x, x * 10, x * 100))
```

Reply:
Oh i didn't realize the doc were tested !
That is pretty great actually :)
Do you have any pointers on what you're using to achieve that ?
I would be interested to reuse it
Thanks !